internal/schemautil/service.go

```diff
@@ -774,9 +811,9 @@ func copyServicePropertiesFromAPIResponseToTerraform(
 		return fmt.Errorf("cannot set `components` : %w", err)
 	}
 
-	// Handle read_replica service integrations
-	readReplicaIntegrations := getIntegrationsForTerraform(s.ServiceIntegrations, service.IntegrationTypeReadReplica)
-	if err := d.Set("service_integrations", readReplicaIntegrations); err != nil {
+	// Handle service integrations
+	integrations := flattenIntegrations(s.ServiceIntegrations, getBootstrapIntegrationTypes(serviceType)...)
+	if err := d.Set(serviceIntegrationsKey, integrations); err != nil {
 		return err
 	}
```

Comment:
It sets back only `read_replica` and `disaster_recovery`. Said that, we should validate `integration_type` user input [here](https://github.com/aiven/terraform-provider-aiven/pull/1867/files#diff-6531b5312c16e9c85a338320bf80d70d135f7057d0d748540534d5e76ed6bc36R83), and, basically disable `service_integrations` field for the most of the service types. Only `mysql` and `pg` types can define integrations there.

Reply:
So before this change it never read things back https://github.com/aiven/terraform-provider-aiven/commit/f7a7a5ceb35bb2316950b5e7b4de20f211439faf#diff-6531b5312c16e9c85a338320bf80d70d135f7057d0d748540534d5e76ed6bc36R789
I wonder if I should keep the validation or remove it. It is probably a breaking change, but it makes 100% sense to me.